src/screens/TopicMemberScreen/index.js

```diff
@@ -208,4 +207,4 @@ const TopicMemberScreen = () => {
     </SafeAreaProvider>
   );
 };
-export default withInteractionsManaged(TopicMemberScreen);
+export default TopicMemberScreen;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `withInteractionsManaged` HOC could potentially impact the user experience if it was used to manage interactions such as animations or gestures. If these interactions are not managed elsewhere, you might want to reconsider this change.
```diff
- export default withInteractionsManaged(TopicMemberScreen);
+ export default TopicMemberScreen;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->